### PR TITLE
feat: support PR comments for push events

### PR DESCRIPTION
**Pull request by micnncim**
Since GitHub no longer shows commit comments in the pull request timeline [^1], it needs to find a PR associated with a commit in a push event and then post a PR comment if found in order to show comments in the pull request timeline.

See https://github.com/suzuki-shunsuke/tfcmt/issues/387 for more details.

[^1]: github.blog/changelog/2022-08-04-commit-comments-no-longer-appear-in-the-pull-request-timeline

### CODE REVIEW

**Comment by micnncim** [comment]
@suzuki-shunsuke

**Comment by suzuki-shunsuke** [comment]
Thank you for your contribution!

File: pkg/notifier/github/commits.go [hunk 1]
```diff
@@ -9,13 +9,21 @@ import (
...
+)
+
+func (g *CommitsService) PRNumber(ctx context.Context, sha string, state PullRequestState) (int, error) {
 	prs, _, err := g.client.API.PullRequestsListPullRequestsWithCommit(ctx, sha, nil)
 	if err != nil {
 		return 0, err
 	}
 	for _, pr := range prs {
-		if pr.GetState() != "closed" {
+		if pr.GetState() != string(state) {
```
Comment (suzuki-shunsuke) [inline pkg/notifier/github/commits.go:26]: Could you check Pull Request's `UpdatedAt`?

> How to handle when multiple pull requests are associated
> Choose the latest updated pr

https://github.com/suzuki-shunsuke/tfcmt/issues/387#issuecomment-1206353018

I don't know the order of `prs`

Comment (micnncim) [reply pkg/notifier/github/commits.go]: Done: [`d359a60` (#390)](https://github.com/suzuki-shunsuke/tfcmt/pull/390/commits/d359a60fbd9a02da655b75864565efa3b149e9ca)

**Review by suzuki-shunsuke — APPROVED** [review]
Thanks!

**Comment by micnncim** [comment]
@suzuki-shunsuke Thanks for your quick review! When are planning to release a new version?

**Comment by suzuki-shunsuke** [comment]
I have already released v3.4.0.
https://github.com/suzuki-shunsuke/tfcmt/releases/tag/v3.4.0

**Comment by micnncim** [comment]
I missed the [release](https://github.com/suzuki-shunsuke/tfcmt/releases/tag/v3.4.0). Thanks a lot again.

**Pull request merged**